colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -193,6 +193,7 @@
         "run = wandb.init(project=\"artifact-basics\")\n",
         "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to\n",
         "artifact.add_file(local_path=\"/content/sample_data/california_housing_test.csv\", name=\"new_file\")\n",
```

Comment:
I think we can remove the name argument - this is an optional argument that I don't see users use too often